include/util/multi_level_partition.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef OSRM_UTIL_MULTI_LEVEL_PARTITION_HPP
+#define OSRM_UTIL_MULTI_LEVEL_PARTITION_HPP
+
+#include "util/typedefs.hpp"
+#include <cstdint>
+
+namespace osrm
+{
+namespace util
+{
+
+using LevelID = std::uint8_t;
+using CellID = std::uint32_t;
+
+// Mock interface, can be removed when we have an actual implementation
+class MultiLevelPartition
```

Comment:
No runtime polymorphism will be removed, see comment. This is just during implementation. I didn't want to couple building this class and the actual MultiLevelPartition.